# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cryptsetup/hotfix-zero-regs.patch.ia64
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cryptsetup-2.8.0/lib/crypto_backend/memutils.c.vanilla	2025-08-08 14:13:44.416011562 +0200
+++ cryptsetup-2.8.0/lib/crypto_backend/memutils.c	2025-08-08 14:13:13.756260389 +0200
@@ -12,7 +12,7 @@
 #if defined __has_attribute
 #  if __has_attribute (zero_call_used_regs)
 #    undef ATTR_ZERO_REGS
-#    define ATTR_ZERO_REGS __attribute__ ((zero_call_used_regs("used")))
+#    define ATTR_ZERO_REGS // __attribute__ ((zero_call_used_regs("used")))
 #  endif
 #endif
 
